### DISCUSSION TITLE
volume discount pricing

### DISCUSSION THREAD

**Discussion by egant** [saleor/saleor#11871] [Q&A] [answered]
hello everyone,

Is there a recommended way to implement volume discount pricing on a product?

I'm interested in applying threshold discounts to a product, but when using the dashboard I cannot seem to achieve this with a product variant or a sale or a voucher.

**Comment by timuric** [comment, marked as answer]
@egant currently you can achieve that by overriding the price of the item in the cart https://docs.saleor.io/docs/3.x/developer/checkout#setting-custom-prices-for-checkout-items
We are also working on a new promotion engine that would allow creating such logic natively from the dashboard.
